### PR TITLE
gateway test suite: 2/n

### PR DESCRIPTION
**Pull request by okjodom**
test gateway balance

### CODE REVIEW

File: gateway/ln-gateway/tests/integration_tests.rs [hunk 1]
```diff
@@ -59,17 +59,65 @@ async fn gatewayd_shows_info_about_all_connected_federations() {
...
+    let (_, rpc, fed1, fed2, _) = fixtures::fixtures(None).await;
 
-    Ok(())
+    let id1 = fed1.connection_code().id;
+    let id2 = fed2.connection_code().id;
+
+    connect_federations(&rpc, &[fed1, fed2]).await.unwrap();
+
+    let bal1 = rpc
+        .get_balance(BalancePayload { federation_id: id1 })
```
Comment (okjodom) [inline gateway/ln-gateway/tests/integration_tests.rs:74]: @jkitman, can we support balance queries and assertions with just the dummy module?
this gateway test fails with an error that signals the lack of mint module

File: gateway/ln-gateway/tests/integration_tests.rs [hunk 2]
```diff
@@ -59,17 +59,65 @@ async fn gatewayd_shows_info_about_all_connected_federations() {
...
+async fn gatewayd_allows_deposit_to_any_connected_federation() {
+    let (_, rpc, fed, _, bitcoin) = fixtures::fixtures(None).await;
 
-    Ok(())
+    let id = fed.connection_code().id;
+
+    connect_federations(&rpc, &[fed]).await.unwrap();
+
+    let addr = rpc
+        .get_deposit_address(DepositAddressPayload { federation_id: id })
```
Comment (okjodom) [inline gateway/ln-gateway/tests/integration_tests.rs:100]: same here, except now we need the wallet module

File: gateway/ln-gateway/tests/integration_tests.rs (hunk 1 above)
Comment (jkitman) [reply gateway/ln-gateway/tests/integration_tests.rs:74]: The gateway needs to be converted to use the NG client.  I suggest focusing on this before we can get these tests to all pass.

File: gateway/ln-gateway/tests/integration_tests.rs (hunk 2 above)
Comment (jkitman) [reply gateway/ln-gateway/tests/integration_tests.rs:100]: We will need to use the wallet module for testing deposits, but again better to wait for the NG client.

File: gateway/ln-gateway/tests/integration_tests.rs (hunk 1 above)
Comment (okjodom) [reply gateway/ln-gateway/tests/integration_tests.rs:74]: ack.

File: gateway/ln-gateway/tests/integration_tests.rs (hunk 2 above)
Comment (okjodom) [reply gateway/ln-gateway/tests/integration_tests.rs:100]: ack.

**Comment by okjodom** [comment]
closed in favor of a new test suite coming in with gateway #2600

**Pull request closed without merging**